static/js/alert.js

```diff
@@ -818,12 +819,49 @@ function alertDetailsFunctions() {
     });
 }
 
-function createAlertFromLogs(queryLanguage, query, startEpoch, endEpoch) {
+function createAlertFromLogs(queryLanguage, searchText, startEpoch, endEpoch, filterTab) {
+    // Focus on the alert rule name input field
     $('#alert-rule-name').focus();
-    $('#query').val(query);
-    $(`.ranges .inner-range #${startEpoch}`).addClass('active');
+
+    // Set the query language and mark the appropriate option as active
+    $('#query-language-btn span').html(queryLanguage);
+    $('.query-language-option').removeClass('active');
+
+    if (queryLanguage === 'SQL') {
```

Comment:
We do need to add a condition for `query language` as we only support Splunk QL in alerts.